src/file-storage.module.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { AutomapperModule } from '@automapper/nestjs';
+import { ConfigModule } from '@nestjs/config';
+import { FileStorageModule } from './file-storage.module';
+import { FileStorageType } from './enum/file-storage-type.enum';
+import { Test } from '@nestjs/testing';
+import { classes } from '@automapper/classes';
+import { storageConfig } from './config';
+
+describe('FileStorageModule', () => {
+  let fileStorageModule: FileStorageModule;
+
+  it('Should be defined with mock configuration', async () => {
+    const mockS3Config = () => ({
+      aws: {
+        accessKeyId: 'mock-access-key',
+        secretAccessKey: 'mock-secret-key',
+      },
+      storageService: FileStorageType.S3,
+    });
+
+    const app = await Test.createTestingModule({
+      imports: [
+        AutomapperModule.forRoot({ strategyInitializer: classes() }),
+        ConfigModule.forRoot({
+          isGlobal: true,
+          load: [mockS3Config],
+        }),
+        FileStorageModule,
+      ],
+    }).compile();
+
+    fileStorageModule = app.get<FileStorageModule>(FileStorageModule);
+    expect(fileStorageModule).toBeDefined();
+  });
+
+  it('Should be defined with env configuration', async () => {
+    process.env.STORAGE_SERVICE = FileStorageType.S3;
```

Comment:
Env should not be mocked directly, it may affect the tests. ConfigService should be mocked directly.